### PR TITLE
Fix getch on windows

### PR DESCRIPTION
**Pull request by pakrym**
Fixes https://github.com/nvbn/thefuck/issues/378

### CODE REVIEW

File: thefuck/main.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 from warnings import warn
 from pprint import pformat
 import sys
+import win_unicode_console
+win_unicode_console.enable() #https://github.com/tartley/colorama/issues/32
```
Comment (nvbn) [inline thefuck/main.py]: Is the issue related here?

File: thefuck/ui.py [hunk 2]
```diff
@@ -6,7 +6,19 @@
...
-    from msvcrt import getch
+    import msvcrt
+    def getch():
+        ch = msvcrt.getch()
+        if ch in (b'\x00', b'\xe0'):  # arrow or function key prefix?
+            ch = msvcrt.getch()  # second call returns the actual key code
+
+        if ch == b'\x03':
+            raise KeyboardInterrupt
+        if ch == b'H':
```
Comment (nvbn) [inline thefuck/ui.py]: `msvcrt.getch` returns `H` for `↑` and `P` for `↓`?

**Comment by nvbn** [comment]
I've made a few changes, please check branch `pakrym-master`.

File: thefuck/ui.py (hunk 2 above)
Comment (pakrym) [inline thefuck/ui.py]: Yes. It returns two chars one \xe0 and one of those.

File: thefuck/main.py (hunk 1 above)
Comment (pakrym) [inline thefuck/main.py]: Yes, you need to init `win_unicode_console` before importing `colorama`

**Comment by pakrym** [comment]
You moved `win_unicode_console.enable()` call after importing `colorama` they fixed the issue (https://github.com/tartley/colorama/issues/32) but this change is not in `colorama-0.3.3` which `thefuck` uses.

**Comment by nvbn** [comment]
@pakrym they said:
> Since win_unicode_console replaces the streams in order to support Unicode, win_unicode_console.enable has to be called before colorama.init so everything works as expected.

So `win_unicode_console.enable` call is before `colorama.init`.

Also I've updated code in branch `pakrym-master`, please check is it works on windows correctly =)

**Comment by pakrym** [comment]
I meant other part of it:
> As of colorama v0.3.3, there is an early binding issue (https://github.com/tartley/colorama/issues/32), so win_unicode_console.enable has to be called even before importing colorama

**Comment by nvbn** [comment]
@pakrym ok, now it calls `win_unicode_console.enable` before import.

Is it work ok?

**Comment by nvbn** [comment]
Checked by myself, seems working.

**Pull request merged**